test/integration/hotrestart_test.sh

```diff
@@ -4,13 +4,13 @@ set -e
 
 [[ -z "${ENVOY_BIN}" ]] && ENVOY_BIN="${TEST_RUNDIR}"/source/exe/envoy-static
 
-# TODO(htuch): Clean this up when Bazelifying the hot restart test below. At the same time, restore
-# some test behavior lost in #650, when we switched to 0 port binding - the hot restart tests no
-# longer check socket passing. See #654.
+# TODO(htuch): Clean this up when Bazelifying the hot restart test below.
```

Comment:
Can you follow up with @htuch and see that this TODO is about? I'm wondering if it's still relevant. If so, can you add a bit more color? (I know not really related to your change but while I'm noticing).